app/views/home/index.slim

```diff
@@ -1,6 +1,6 @@
-= react_component("Header", nil, { prerender: false })
+= react_component_helper(component: "Header", prerender: true)
```

Comment:
Is there any situation where you won't want prerender? Or, why don't you make the prerender have a default true, and only pass an argument if you don't want it. (keeping the rest of the logic)

Reply:
Yes good idea, the point here is that server-rendering with rails during development. It's hard to debug. We want to force developers to NOT use pre-rendering and only use it during the staging phase.

We want pre-rendering because it's a static page, so people that hate javascript and turn it off can still see the page. Also, it increases a lot of SEO related metrics like time until interactive and such, as the page comes with the HTML already outputted, we don't need to mount components all over again.

Reply:
But still, I think prerender should be OPT-IN.

Even though most of our components will be prerendered, I still think it's good to specify that. Because normally you don't use prerendering in most situations (unless you use a static generator for a project)

Reply:
As you wish 👨‍⚕️ I'm fine with any of them. 
And what if we had an `react_static_component_helper` ? Much readable, such declarative 😄 

It's fine either ways, so I'm approving it

Reply:
I think this way is better, it's just an alternative with the shorter syntax to the original helper.